scripts/minikube.sh

```diff
@@ -154,7 +172,20 @@ EXTRA_CONFIG="${EXTRA_CONFIG} --extra-config=kubelet.resolv-conf=${RESOLV_CONF}"
 #extra Rook configuration
 ROOK_BLOCK_POOL_NAME=${ROOK_BLOCK_POOL_NAME:-"newrbdpool"}
 
+kube_latest_version=$(curl -L https://storage.googleapis.com/kubernetes-release/release/stable.txt 2> /dev/null)
+if [[ -z "${kube_latest_version}" ]]; then
+    # skip: getting latest version failed, continue with what we have
+    if [[ "${KUBE_VERSION}" == "latest" ]]; then
+        echo "ERROR: failed getting kube version"
+        exit 1
+    fi
+else
+    # update the version string from latest with the real version
+    KUBE_VERSION=${kube_latest_version}
```

Comment:
Why? This should not be needed.

Reply:
There are other places where KUBE_VERSION is used, if KUBE_VERSION="latest" then it is just a string indicating us that we need latest kube and is not useful.